### ISSUE TITLE
Package for DGSEA

### ISSUE THREAD

**Issue by llrs** [JamesJoly/DGSEA#1]
Hi, I've seen [the preprint](https://www.biorxiv.org/content/10.1101/860460v1) and I would like to know if you will release DGSEA as a package for easier usage of the methodology. 

I looked at the source code but I found quite hard to know where are the improvements made with DGSEA. Also I'm not sure if I could use the code as it lacks a license. 

Many thanks

**Comment by JamesJoly** [comment]
Hi Lluis,

Thanks for reaching out, we're excited that there seems to be interest in
DGSEA!

Yes, we do plan on releasing an R package. It's a work in progress and I
hope to have it out by the time the paper is accepted for publication
(ideally sooner).

Meanwhile, since there appears to be interest. I'm considering writing a
quick guide on how to use the source code, would that be helpful?

Best,
James
-----------------------------------------------------------------------------------------------
Ph.D Candidate
Mork Family Department of Chemical Engineering and Materials Science
University of Southern California
Office: HED 111
------------------------------------------------------------------------------------------------

**Comment by llrs** [comment]
Yes, a guide would help but I'm not sure it will be enough because the code doesn't run on my computer.

**Comment by JamesJoly** [comment]
My apologies for the delay. I have just uploaded a new version of the source code (v17) to make it more user friendly. There is also an example and a brief description in the README file. The example should reproduce Figures 2B-C of the manuscript. 

To use the script please update lines 7-14 to reflect your path, input file names, and whether or not you want to generate mountain plots. 

Let me know if you have any questions. We're still planning on making it into an R package, but that will take a bit of time :)
